### DISCUSSION TITLE
Very Slow

### DISCUSSION THREAD

**Discussion by nishanthc** [vzhd1701/enex2notion#13] [Q&A] [answered]
Thanks for developing this. I was just wondering why the process is so slow and if there is any way of speeding this up?

**Comment by vzhd1701** [comment, marked as answer]
It is slow because notes are uploaded block-by-block, where each block is roughly equal to one paragraph. The only way to speed it up would be making it run in parallel, but I am afraid that would result in rate limits from the Notion server.
